builds.yml

```
@@ -137,6 +138,7 @@ features:
   keyring-snaps:
     assets:
       - ./{app,shared,ui}/**/keyring-snaps/**
+  bitcoin:
   solana:
     assets:
       - ./{app,shared,ui}/**/solana/**
```

Shouldn't bitcoin have its own files ?
```suggestion
  bitcoin:
    assets:
      - ./{app,shared,ui}/**/bitcoin/**
  solana:
    assets:
      - ./{app,shared,ui}/**/solana/**
```

---

In reality there isn't a `solana/` folder since it wasn't needed. So unless it's mandatory to have this entry here, I would say we can remove both (for btc sol) cc @GuillaumeRx

---

I feel like it's still valuable to add/keep, it makes things easier to split as the folder acts like a code fence.

---

None of those paths exist (not for Bitcoin at least)